sv/ypbind/check

```diff
@@ -1,3 +1,4 @@
 #!/bin/sh
 
-timeout 1 ypwhich
+date > /tmp/checkme
```

Comment:
What happens with /tmp/checkme?  Is it just collecting data on when ./check was run?